libnmstate/ifaces/vrf.py

```diff
@@ -46,8 +49,10 @@ def is_virtual(self):
 
     def pre_edit_validation_and_cleanup(self):
         super().pre_edit_validation_and_cleanup()
-        if self.is_up:
+        if self.is_up and self.is_desired:
```

Comment:
Should we validate and cleanup if it is only changed but not desired? What fo you think?

Reply:
Changed to `if self.is_up and (self.is_desired or self.is_changed):`

Reply:
Thanks!